fhir-search/src/main/java/com/ibm/fhir/search/parameters/ParametersUtil.java

```diff
@@ -40,6 +48,10 @@
     public static final String FHIR_DEFAULT_SEARCH_PARAMETERS_FILE = "search-parameters.json";
     public static final String FROM_STEAM = "from_stream";
 
+    private static final Set<String> ALL_RESOURCE_TYPES = ModelSupport.getResourceTypes(true).stream()
```

Comment:
is abstract needed here? I think so since we have some FHIRPath expressions which are `^Resource.*$`

**just double checking**

Reply:
yes, we want to gather search parameters with a base of Resource (or DomainResource / other abstract types in future)